README.md

```diff
@@ -39,10 +39,17 @@ def mystery2(n)
 end
 ```
 
-- What is mystery2(123)?
-- What is mystery2(9005)?
-- What is mystery2(-123)?
+- What is mystery2(123)? 6
+- What is mystery2(9005)? 14
+- What is mystery2(-123)? -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
+- It looks like this method is created in order to add all the digits together of each number.  Meaning a number entered 123 should result in 6.  To account for negative numbers, we could create an earlier if statement:
+
+if n < 0 && n >= -10
+  return n
+else
+  return (n%10) - mystery2(n/10)
+end
```

Comment:
@sudocrystal I have tried to mess around with this, after remembering that mod doesn't work well with negative numbers.  I remember in class that you did something to retain whether the number was negative or positive, then multiply the result by that before returning.  I can't seem to get that to work (not sure where to put it with the multiple returns).  Do you have a code snipped I could look at or a suggestion?  Thank you!